### PR TITLE
Reinstate no sync

### LINKED ISSUE #954: --sync=false option for `convox start` no longer works
It looks like there was a regression in this feature in CLI version `20160726184421` (possibly earlier, haven't confirmed). If you run `convox start --sync=false` files are still synced.

### PR DESCRIPTION
**Pull request by awsmsrc**
fixes #954 

for your viewing pleasure @mattmanning and @MiguelMoll

### CODE REVIEW

File: cmd/convox/start.go [hunk 1]
```diff
@@ -18,18 +18,18 @@ import (
 func init() {
 	stdcli.RegisterCommand(cli.Command{
 		Name:        "start",
-		Description: "start an app for local development",
+		Description: "Start an app for local development",
```
Comment (ddollar) [inline cmd/convox/start.go]: these are lowercase everywhere else

File: cmd/convox/start.go [hunk 2]
```diff
@@ -84,7 +84,9 @@ func cmdStart(c *cli.Context) error {
 	}
 
 	cache := !c.Bool("no-cache")
-	r := m.Run(dir, app, cache)
+	sync := c.Bool("sync")
```
Comment (ddollar) [inline cmd/convox/start.go]: Let's line this up with the `no-cache`. This should be invoked with `convox start --no-sync`

Comment (awsmsrc) [inline cmd/convox/start.go]: yeah i thought that was odd, but i went with what was there originally.
Will change it over for consistency

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;xtXM7yVU&#x2F;435-no-sync-fixes"> no-sync fixes</a>

**Comment by ddollar** [comment]
Looking good. Please satisfy the Hound and rebase and we'll get this in the next release.

**Comment by awsmsrc** [comment]
@ddollar done done and done

**Pull request closed without merging**